app/views/help/eng/publishing.markdown

```diff
@@ -140,7 +140,7 @@ To get started, find a set of documents you wish to embed -- either by selecting
 Once you're comfortable with your settings, preview the embedded document set. If the preview looks good, copy and paste the HTML embed code. Here's an example of what the embed code should look like:
 
     <div id="DC-search-projectid-8-epa-flouride"></div>
-    <script src="//assets.documentcloud.org/embed/loader.js"></script>
+    <script src="//assets.sourceafrica.net/embed/loader.js"></script>
```

Comment:
@4bic We currently don't have an assets.sourceafrica.net running. We still uses DC's CDN.